### PR TITLE
Reshuffle and discard buttons for draw simulator

### PR DESCRIPTION
**Pull request by eric-parsons**
This is a small change to add a new "Reshuffle Selected" button to the draw simulator, similar to [arkhamdb.com](http://arkhamdb.com) (the code was adapted from there). There are a whole bunch of cards in the pool for which this is useful, e.g. Master of the Forge, The Eagles Are Coming!, Mustering the Rohirrim, Timely Aid, etc. Decks that rely on these cards for consistency are difficult to test in the simulator without this functionality.

I chose not to copy the "Redraw Selected" button seen on arkhamdb, since the same effect can be easily accomplished by reshuffling and then drawing new cards, and there aren't many cards that could benefit from such a shortcut other than maybe Drinking Song. Instead, I added a "Discard Selected" button. This is handy for a bunch of cards in the pool, e.g. Erestor (both versions), A Very Good Tale, Daeron's Runes, etc., and avoids having to mentally keep track of what's in your hand and what isn't.

Hopefully you find these changes useful!

### CODE REVIEW

File: src/AppBundle/Resources/views/Builder/draw-simulator.html.twig [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
+    <tbody>
+        <tr>
+            <td class="text-center" title="Click to draw; Shift-click to reset and draw">
+                <div class="btn-group">
+                    <button class="btn btn-default btn-sm" disabled="disabled">Draw:</button>
+                    <button class="btn btn-default btn-sm" data-command="1">1</button>
+                    <button class="btn btn-default btn-sm" data-command="2">2</button>
+                    <button class="btn btn-default btn-sm" data-command="6">6</button>
+                    <button class="btn btn-default btn-sm" data-command="all">all</button>
+                    <button class="btn btn-default btn-sm" data-command="reshuffle" disabled="disabled">Reshuffle Selected</button>
```
Comment (eric-parsons) [inline src/AppBundle/Resources/views/Builder/draw-simulator.html.twig]: This line and the next are new, the rest is just whitespace fixes (the file contained mixed tabs & spaces).

**Comment by eric-parsons** [comment]
@seastan, any interest in this? If so I'll fix the merge conflicts.

**Comment by seastan** [comment]
Yes, still interested, I was just working on another feature. If you can resolve the conflicts I'll take a look.

**Comment by eric-parsons** [comment]
> Yes, still interested, I was just working on another feature. If you can resolve the conflicts I'll take a look.

Great! Done.

**Pull request merged**